docs/UserGuide.md

```diff
@@ -60,204 +60,235 @@ reliably than traditional GUI based apps.
 
 **Warning: `WARD_NAME` is case-sensitive.** `block B ward 2` will refer to a different ward from `Block b Ward 2`
 
-### Format: `addward n/WARD_NAME`
+**Format: `addward n/WARD_NAME`**
 `WARD_NAME` must be unique and cannot be the same as any existing name. This field is case-sensitive.
 
-### Examples:
+**Examples:**
 `addward n/block B ward 2` will create a new empty ward called block B ward 2\
 `addward n/block C ward 1` will create a new empty ward called block C ward 1
 
 [Go back to [Table of Contents](#table-of-contents)]
 
 ---
-## Listing all patients: lspat
+## Listing all patients: lspat (Is WARD_NAME still a valid argument?)
 Lists all the patients’ names and corresponding patient ID, displaying any tags attached to them and showing the ward they are in.
 
 **Warning: `WARD_NAME` is case-sensitive.** `block B ward 2` will refer to a different ward from `Block b Ward 2`
 
-### Format: `lspat [w/WARD_NAME]`
+**Format: `lspat [w/WARD_NAME]`**
 `WARD_NAME` can be specified to only list patients in the specified ward. This field is case-sensitive. This is an optional parameter.
 
-### Examples:
+**Examples:**
 `lspat` will list all patients in each ward on the GUI. Every ward will be displayed in order, with all patients in each ward.\
 `lspat w/block B ward 2` will display the list of patients that are assigned to block B ward 2 in the GUI
 
 [Go back to [Table of Contents](#table-of-contents)]
 
 ---
-## Listing staff members: lsstf
+## Listing staff members: lsstf (Is WARD_NAME still a valid argument?)
 
 Lists staff members’ names and the name of the ward they are assigned to.
 
 **Warning: `WARD_NAME` is case-sensitive.** `block B ward 2` will refer to a different ward from `Block b Ward 2`
 
-### Format: `lsstf [w/WARD_NAME]`
+**Format: `lsstf [w/WARD_NAME]`**
 `WARD_NAME` can be specified to only list staff in the specified ward. This is an optional parameter.
 
-### Examples:
+**Examples:**
 `lsstf` will list all staff assigned to each ward on the GUI. The lists will be grouped according to ward, and each staff
 can appear more than once if they are assigned to more than 1 ward.\
 `lsstf w/block A ward 1` will list all staff assigned to block A ward 1 on the GUI.
 
 [Go back to [Table of Contents](#table-of-contents)]
 
 ---
-## Listing a patient’s prescriptions: lspresc
-Lists the prescriptions and instructions list for a patient in numbered list form for the patient specified by `ID_NUMBER`.
+## Finding a person: find (Should still be right, delete this note if it is)
+
+Finds all persons with names containing any of the specified keywords and displays them on the GUI.
 
-### Format: `lspresc id/ID_NUMBER`
+**Warning: `NAME` is case-insensitive.** Using an argument `Alice` will match names `aLice`, `alice`, `ALICE` etc.
 
-### Examples:
-`lspresc id/A0123456789B` will display the list of tasks and prescriptions of patient with ID A0123456789B on the GUI
+**Format: `find NAME`**
+
+**Examples: `find Alice Tan`**
```

Comment:
I think it will be good to add a note here that this will find all that have names that contain Alice or Tan.